src/client/v2/indexer/models/types.ts

```diff
@@ -3,6 +3,7 @@
  */
 
 /* eslint-disable no-use-before-define */
+import { ensureBigInt, ensureSafeInteger } from '../../../../utils/utils.js';
```

Comment:
do we really want to re-generate v2 client models?

Reply:
Yes this brings some improvements. Before every integer field was `number | bigint`, which is not useful because you'd need to cast to one of them before actually comparing or operating on them